sdk/go/common/resource/urn/urn.go

```diff
@@ -201,5 +201,26 @@ func (urn URN) Rename(newName string) URN {
 
 // Returns a new URN with an updated stack part
 func (urn URN) RenameStack(stack tokens.StackName) URN {
```

Comment:
Given the easy typo on this can we have some sanity tests in urn_test that these don't muck up any other aspects of the URN except for the field being changed.

Reply:
Yup, good idea.  Done :heavy_check_mark: